src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,26 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Method login was called with login: " + login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
+            logger.error("Authentication failed for login: " + login);
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("User logged in successfully with login: " + login);
         return user;
     }
 
     private User findByLogin(String login) {
-        User user = new User(login, "1234");
-        // this user identifier should be set by DB. We will use dummy data for this example
-        user.setUserId(2L);
-        return user;
+        logger.info("Searching for user with login: " + login);
+        throw new UnsupportedOperationException("Implement database connection to retrieve user");
```

Comment:
The `findByLogin` method is currently not implemented and throws an `UnsupportedOperationException`. You need to implement this method to connect to a database or data source to retrieve the user information based on the login. This is crucial for the functionality of the authentication service.